notifications.bs

```diff
@@ -704,20 +773,75 @@ getter must return the <a lt="concept notification">notification</a>'s
 getter must return a <a>structured clone</a> of
 <a lt="concept notification">notification</a>'s <a>data</a>.
 
+<p>The <dfn attribute dfn-for=Notification><code>actions</code></dfn>
+attribute's getter must return the result of the following steps:
+
+<ol>
+  <li><p>Let <var>frozenActions</var> be an empty list of type
+  {{NotificationAction}}.
+
+  <li><p>For each <var>entry</var> in the
+  <a lt="concept notification">notification</a>'s list of <a>actions</a>,
+  perform the following steps:
+
+  <ol>
+    <li><p>Let <var>action</var> be a new {{NotificationAction}}.
+
+    <li><p>Set <var>action</var>'s {{NotificationAction/action}} to
+    <var>entry</var>'s <a>action name</a>.
+
+    <li><p>Set <var>action</var>'s {{NotificationAction/title}} to
+    <var>entry</var>'s <a lt="action title">title</a>.
+
+    <li><p>Call <a lt="freeze">Object.freeze</a> on <var>action</var>.
```

Comment:
I think you need to say here that you are using the initial value of `Object.freeze` so it cannot possibly be overridden from script. There should probably also be some XXX type of comment in the source here and a bug on IDL since it's somewhat strange to invoke JavaScript on what are effectively IDL constructs.